src/inmanta/env.py

```diff
@@ -173,7 +173,9 @@ def _run_command_and_log_output(
         try:
             output = subprocess.check_output(cmd, stderr=stderr, env=env)
         except CalledProcessError as e:
-            LOGGER.error("%s: %s", cmd, e.output.decode())
+            err: str = e.stderr.decode()
+            msg: str = err if err else e.output.decode()
+            LOGGER.error("%s: %s", cmd, msg)
```

Comment:
This has nothing to do with the warnings, I just noticed for the nth time that this error message wasn't all that useful to determine the actual cause so I thought I'd make an attempt to fix it.

Reply:
Nice!